api/src/pcapi/routes/backoffice/filters.py

```diff
@@ -1701,6 +1701,50 @@ def format_finance_incident_type(incident_kind: finance_models.IncidentType) ->
             return incident_kind.value
 
 
+def format_notice_type(notice_type: offerers_models.NoticeType) -> str:
+    match notice_type:
+        case offerers_models.NoticeType.UNPAID_AMOUNT_NOTICE:
+            return "Avis d'impayé"
+        case offerers_models.NoticeType.REMINDER_LETTER:
+            return "Lettre de relance"
+        case offerers_models.NoticeType.BAILIFF:
+            return "Huissier de justice"
+        case _:
+            return notice_type.value
+
+
+def format_notice_status_motivation(notice_motivation_type: offerers_models.NoticeStatusMotivation | None) -> str:
+    match notice_motivation_type:
+        case offerers_models.NoticeStatusMotivation.ALREADY_PAID:
+            return "Déjà payé"
+        case offerers_models.NoticeStatusMotivation.REJECTED:
+            return "Paiement rejeté"
+        case offerers_models.NoticeStatusMotivation.NO_LINKED_BANK_ACCOUNT:
+            return "Pas de CB rattaché"
```

Comment:
CB = coordonnées bancaires
```suggestion
            return "Pas de CB rattachées"
```

Reply:
ok, y'a pas la place pour l'écrire en entier ? :)